docs/docs/how-tos/nebari-aws.md

```diff
@@ -105,24 +105,42 @@ environment variables have been properly set. It is time to initialize and deplo
    mkdir nebari-aws && cd nebari-aws
    ```
 
-2. Executing the command below will generate a basic config file with an infrastructure based on **AWS**, with project name `projectname`, endpoint domain `domain`, and with the authentication mode set to **password**.
+2. Executing the command below will generate the set of questions that needs the suitable values to generate the qhub-config file with an infrastructure based on based on **AWS**, with project name `projectname`, endpoint domain `domain`, and with the authentication mode set to **password**.
 
 ```bash
-qhub init aws --project projectname \
+   nebari init --guided-init
+   ```
+![A representation of the output generated when Nebari init guided-init command is executed.](/img/how-tos/nebari-guided-init-aws.png)
+
+![A representation of the output generated when Nebari init guided-init command is executed.](/img/how-tos/nebari-guided-init-end-part.png)
+
+
+3. (Optional) Executing the command below will generate a basic config file with an infrastructure based on **AWS**, with project name `projectname`, endpoint domain `domain`, and with the authentication mode set to **password**.
+
+```bash
+nebari init aws --project projectname \
 	  --domain domain \
 	  --auth-provider password
 ```
```

Comment:
So I can run `nebari init --guided-init` and then `nebari init -- blah blah`?

I think this should be a tip like "if you prefer not using the guided init command you can use......"